flow/instrumentation.cc

```diff
@@ -17,6 +18,8 @@ static const size_t kMaxFrameMarkers = 8;
 Stopwatch::Stopwatch() : start_(fml::TimePoint::Now()), current_sample_(0) {
   const fml::TimeDelta delta = fml::TimeDelta::Zero();
   laps_.resize(kMaxSamples, delta);
+  cache_dirty_ = true;
+  prev_drawn_sample_ = 0;
```

Comment:
This is actually meant to be the index of the previously drawn sample, not a count. This is so that if the overlay is turned off and then on again, the Stopwatch knows where to erase the red/green vertical bar previously drawn. prev_drawn_sample_index_ is probably a more fitting name.